byway/src/main/webapp/base.js

```diff
@@ -13,17 +13,21 @@ function setProgressBar(pageNumber){
     items[i].className = 'active';
   }
   let tripKey = getTripKeyFromUrl();
-  document.getElementById('choose-destinations-progress-bar').href = configureTripKeyForPath(tripKey, '/destinations.html');
-  document.getElementById('choose-interests-progress-bar').href = configureTripKeyForPath(tripKey, '/interests.html');
-  document.getElementById('view-route-progress-bar').href = configureTripKeyForPath(tripKey, '/routepage.html');
+  document.getElementById('choose-destinations-progress-bar').href = 
+configureTripKeyForPath(tripKey, '/destinations.html');
+  document.getElementById('choose-interests-progress-bar').href = 
+configureTripKeyForPath(tripKey, '/interests.html');
```

Comment:
When you put newlines in assignments or after open braces etc, indent the next line farther than the previous.

Reply:
fixed